api/types/constants.go

```diff
@@ -567,6 +567,13 @@ const (
 	// downgraded before being returned to clients on older versions that do not
 	// support one or more features enabled in that resource.
 	TeleportDowngradedLabel = TeleportInternalLabelPrefix + "downgraded"
+
+	// TeleportManagedLabel identifies resources which are managed by Teleport. These resources can
```

Comment:
I'm not 100% clear on what the implications of _managed by Teleport_ and _These resources can be freely modified by Teleport if needed_ are in this context. 

I'm reading it as:
> DANGER! This resource should not be modified bay anyone who is _not_ a Teleport auth server.  Teleport may overwrite/revert/otherwise destroy any changes to this resource at will.

Is that about right? If so, at some point we should add some flashing lights in the UI if you try to edit it.

Reply:
I've updated the wording here -- do you mind taking a look when you get a chance?